src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
```

Comment:
Hi @cynthiainga , I think I use the functional components because it's in the project requirements. Kindly check the screenshot below. And if I miss something or misunderstand something pls let me know.
![Screenshot (58)](https://user-images.githubusercontent.com/111948881/218542172-cb42e33f-25ed-4da2-a2c7-f84ef671d07a.png)